syntax/opam.vim

```diff
@@ -11,6 +11,7 @@ endif
 
 " need %{vars}%
 " env: [[CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"]]
+syn iskeyword 92-122,45
```

Comment:
Why not use `a-z,A-Z,-`?

Reply:
good question, i went directly on ascii numbers, but `a-z,A-Z,-` is also accepted, and intelligible.

Reply:
updated!